obj/marshalSlab.go

```diff
@@ -118,8 +118,8 @@ func _yieldMarshalMachinePtr(row *marshalSlabRow, atl atlas.Atlas, rt reflect.Ty
 		case entry.StructMap != nil:
 			row.marshalMachineStructAtlas.cfg = entry
 			return &row.marshalMachineStructAtlas
-		case entry.MapMorphism != nil:
-			row.marshalMachineMapWildcard.cfg = entry
+		case atl.MapMorphism != nil:
+			row.marshalMachineMapWildcard.morphism = atl.MapMorphism
```

Comment:
This line should revert.  If there *is* an explicit entry in the atlas for this type, we should take it.  (People will probably write that *and* be setting the default at the whole atlas scale at the same time incredibly rarely, but still.)